cointop/config.go

```diff
@@ -507,6 +508,14 @@ func (ct *Cointop) loadAPIChoiceFromConfig() error {
 func (ct *Cointop) loadFavoritesFromConfig() error {
 	log.Debug("loadFavoritesFromConfig()")
 	for k, valueIfc := range ct.config.Favorites {
+		if k == "character" {
+			if favoriteChar, ok := valueIfc.(string); ok {
+				if len(favoriteChar) != 1 {
```

Comment:
I think we should use `utf8.RuneCountInString(favoriteChar)` instead of `len(favoriteChar)` to allow for certain UTF-8 characters

For example, `⭐` which is one rune but composed of 3 bytes

![DeepinScreenshot_select-area_20210929171426](https://user-images.githubusercontent.com/168240/135365425-af4058c1-1254-4212-9e20-b6e845387ad3.png)

![DeepinScreenshot_select-area_20210929171505](https://user-images.githubusercontent.com/168240/135365464-3da4a41d-245f-4ef7-a102-7116d92aac08.png)

Reply:
Good catch!